src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -246,7 +277,8 @@ def _sortable_string_field_specification(field_name):
         }
     }
 
-def set_up_mapping_aip_index(conn):
+
+def set_up_mapping_aip_index(client):
```

Comment:
This function changed in a recent qa/1.x commit, so there'll be a minor merge conflict on rebase.